src/server/middleware/helmet.ts

```diff
@@ -22,6 +21,8 @@ const defaultSrc = [
   'https://tagmanager.google.com',
   'www.googleadservices.com',
   'www.gstatic.com',
+  'https://fonts.gstatic.com',
+  'https://www.google-analytics.com',
```

Comment:
U sure about this one? We have it on L20 already..

Reply:
ah my bad, missed that one. the font is needed though for the GTM preview, the icons is messed up otherwise

Reply:
also needed a few other for the preview mode in optimize apparently